web_interface/blueprints/pages_v3.py

```
@@ -396,7 +395,7 @@ def _load_plugin_config_partial(plugin_id):
                                             config['images'] = config.get('images', []) + new_images
                             except Exception as e:
                                 print(f"Warning: Could not load metadata for {plugin_id}: {e}")
-            except Exception:
+            except Exception:  # nosec B110 - metadata pre-load is optional; schema loads fully below
                 pass  # Will load schema properly below
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)